src/Confluent.Kafka/include_docs.xml

```diff
@@ -0,0 +1,35 @@
+<API>  
+
+<Member name="ISerializer_Configure">  
+    <summary>
+        Configure the serializer using relevant configuration parameter(s) in <paramref name="config" /> (if present)
```

Comment:
'relevant' is a bit vague, is this serializer-specific configuration, or standard librdkafka config that is used by the serializer?